server/src/main/java/org/apache/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -322,6 +328,18 @@ public boolean getReplicateAfterLoadTimeout()
     return replicateAfterLoadTimeout;
   }
 
+  @JsonProperty
+  public Set<String> getUnmanagedNodes()
+  {
+    return unmanagedNodes;
+  }
+
+  @JsonProperty
```

Comment:
Please add a short javadoc mentioning what is the key and value of this map.
We should also document this config.